modules/custom/d_social_media/templates/social_media.html.twig

```diff
@@ -0,0 +1,18 @@
+{% if show %}
+  {{ attach_library('d_social_media/last-element-in-a-row') }}
```

Comment:
Probably it will be more clean to attach this library in SocialMediaBlock.php, by adding `#attached' key to render array